client/pages/apply.jsx

```diff
@@ -78,11 +78,12 @@ class Apply extends React.Component {
                 type="text"
                 required
                 defaultValue={name}
-                pattern="[A-Za-z0-9 ]{3,50}"
+                pattern="[A-Za-z][A-Za-z0-9_\- ]{2,49}"
```

Comment:
Do we want to keep a space here? Isn't that asking for trouble?

Reply:
I don't have a strong opinion on it, but people may want to type out their full names/pseudonyms and include spaces, e.g "Bruce Wayne" or "The Batman". The change makes it so "   " isn't valid anymore, at least.